docs/en/latest/practices/enable-authentication-and-restriction.md

```diff
@@ -27,36 +27,124 @@ Consumers are used for the authentication method controlled by Apache APISIX, if
 
 ## Attributes
 
-### Authentication
+### Authentication methods
 
-#### [`keyAuth`](https://apisix.apache.org/docs/apisix/plugins/key-auth/)
+#### `keyAuth`
 
-Consumers add their key either in a header `apikey` to authenticate their requests.
+Consumers add their key either in a header `apikey` to authenticate their requests. For more information about `keyAuth`, please refer to [APISIX jwt-auth](https://apisix.apache.org/docs/apisix/plugins/key-auth/).
+
+<details>
+  <summary>keyAuth yaml configure</summary>
 
 ```yaml
-keyAuth:
-  value:
-    key: ${key}
+apiVersion: apisix.apache.org/v2beta3
+kind: ApisixConsumer
+metadata:
+  name: ${name}

```

Comment:
I would suggest we keep using the same style when passing variables, cc @navendu-pottekkat @hf400159

Reply:
yes, I've added this to the next edition of the writing guide last week, and I'll be releasing the new style guide this Friday.

Reply:
@AlinsRan Does the `${name}` here refer to a value the user has to set? Or is it somehow a variable they are passing?

In the past we have been using `name: user-configured-name` or some placeholder like that.

// @juzhiyuan @hf400159

Reply:
This is a variable that users can customize. Do I need to follow the style of the past?